### ISSUE TITLE
Add timestamps to bst test output

### ISSUE THREAD

**Issue by dmarvp** [bespoken/bst#515]
**Is your feature request related to a problem? Please describe.**
When errors are reported for bst test with e2e scripts, not all of them are easy to identify. We need timestamps so that it's easier for us to take a look at our server logs and match those errors in order to resolve them.

**Describe the solution you'd like**
Ideally, timestamps would be shown by default just when an error happens and just for the tests that failed.

**Describe alternatives you've considered**
If jest doesn't allow it, we can use a library like this one:
https://www.npmjs.com/package/log-timestamp
but that would affect the whole output. If that's the case it should be done via a configuration property under testing.json `timestamps=true`

**Comment by dmarvp** [comment]
@jperata can you evaluate if this is doable? or propose another alternative?

**Comment by jperata** [comment]
At the moment this is how we print the errors:
``` 
FAIL  test/e2e/e2e.yml (13.462s)
  en-US
    Failing guess the price
      ✕ open Guess the price

  ● en-US › Failing guess the price › open Guess the price

    Error
      Expected value at [prompt] to ==
      	Really?
      Received:
      	welcome to guess the price how many persons are playing today
```
The timestamp indicator can be added at the interaction level (since that is where the specifics of the error are,  we could present it before the locales, in order to show correctly, at the start of the interaction:

``` 
FAIL  test/e2e/e2e.yml (13.462s)
  en-US
    Failing guess the price
      ✕ open Guess the price

  ● [2012-08-23T20:08:32.000Z] > en-US › Failing guess the price › open Guess the price

    Error
      Expected value at [prompt] to ==
      	Really?
      Received:
      	welcome to guess the price how many persons are playing today
```
or before the error description, Jest doesn't provide native timestamps for errors but the change is short once we decide the format

``` 
FAIL  test/e2e/e2e.yml (13.462s)
  en-US
    Failing guess the price
      ✕ open Guess the price

  ● en-US › Failing guess the price › open Guess the price

    Error
      [2012-08-23T20:08:32.000Z] Expected value at [prompt] to ==
      	Really?
      Received:
      	welcome to guess the price how many persons are playing today
```

**Comment by jkelvie** [comment]
My suggestion:
```
Expected value at [prompt] to ==
   Really?
Received:
   welcome to guess the price how many persons are playing today
Timestamp:
   2012-08-23T20:08:32.000Z
```

**Comment by jperata** [comment]
Released on version bespoken-tools@2.0.35

**Comment by MKen-li** [comment]
Marked as verified
Build = bespoken-tools@2.0.35
